### DISCUSSION TITLE
Quarto - Shinylive Extension

### DISCUSSION THREAD

**Discussion by giscus** [quarto-dev/quarto-cli#11412] [Blog]
# Quarto - Shinylive Extension

With Shinylive, you can embed Shiny for Python applications into Quarto documents and run the entire application (including the Python runtime) inside the user’s web browser.

https://quarto.org/docs/blog/posts/2022-10-25-shinylive-extension/

**Comment by joe-chelladurai** [comment]
This is big!

**Comment by AlFontal** [comment]
I am really really happy about this.

**Comment by mcaruel** [comment]
Could you please provide a detailed workflow to run one of the example from the extension repo ? I cannot make it work though I followed the instructions from the README file. The file does explain how to run the apps. Which quarto command to execute in order to be able to preview the document ? `quarto render` creates an `.html` file that I cannot open with my web browser, I get the following error:
" Shinylive uses a Service Worker, which requires either a connection to localhost, or a connection via https." 
Thanks
